docs/state-sync.md

```diff
@@ -0,0 +1,155 @@
+# State Sync
+
+## What is State sync?
+
+The Provenance 1.10.0 release comes with Tenderming Core 0.34 which includes support for state sync. State sync allows a new node to join a network by fetching a snapshot of the application state at a recent height vs. fetching and replaying all historical blocks (which can take days). This can reduce the time needed to sync with the network down to minutes.
```

Comment:
"Tenderming" is typoed. You might also want to put Provenance 1.10.0**+** so that it doesn't easily get out of date.